tree/30_generic_methods/file_from_shared_folder.cf

```diff
@@ -21,7 +21,8 @@
 #
 # @parameter source      Source file (path relative to Rudder shared folder)
 # @parameter destination Destination file (absolute path on the target node)
-# @parameter hash_type   Hash algorithm used to check if file is updated
+# @parameter hash_type   Hash algorithm used to check if file is updated (md5, sha1, sha256), only md5 can be used on cfengine agent
```

Comment:
it's not used on cfengine, so why not simply state that it is not used for cfengine agent ?

Reply:
ping ?